### ISSUE TITLE
Native inheritance/prototype chaining is not working correctly

### ISSUE THREAD

**Issue by Arshia001** [Redfire75369/spiderfire#42]
From the latest commit on master:

```
> new File(['a'], 'b').__proto__.__proto__
{}
```

The second `__proto__` should be `Blob`, but it's `Object` instead. I'm investigating this myself, but I thought it was best to create an issue anyway.

**Comment by Arshia001** [comment]
From the class code generated by ion for `File`:

```rust
impl ::ion::ClassDefinition for File {
	const NAME: &'static str = ::std::stringify!(File);
	fn class() -> &'static ::ion::class::NativeClass {
		Self::__ion_native_class()
	}
	fn constructor() -> (::ion::functions::NativeFunction, ::core::primitive::u32) {
		(Self::__ion_bindings_constructor, 2u32)
	}
	fn functions() -> &'static [::mozjs::jsapi::JSFunctionSpec] {
		Self::__ion_function_specs()
	}
	fn properties() -> &'static [::mozjs::jsapi::JSPropertySpec] {
		Self::__ion_property_specs()
	}
	fn static_functions() -> &'static [::mozjs::jsapi::JSFunctionSpec] {
		Self::__ion_static_function_specs()
	}
	fn static_properties() -> &'static [::mozjs::jsapi::JSPropertySpec] {
		Self::__ion_static_property_specs()
	}
}
```

I think the problem is a missing implementation for `parent_class_info`.

**Comment by Redfire75369** [comment]
Fixed in 6fe9cb2f8af12c410bca362dfd9bebed35e84224.
